x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/authz/store/CompositeRolesStoreTests.java

```diff
@@ -491,6 +491,19 @@ private void trySuccessfullyLoadSuperuserRole(CompositeRolesStore compositeRoles
         final Role role = future.actionGet();
         assertThat(role.names(), arrayContaining("superuser"));
         assertThat(role.application().getApplicationNames(), containsInAnyOrder("*"));
+        assertThat(
```

Comment:
We test this elsewhere (e.g., in `30_superuser`) but seems like an appropriate addition here